config/software/git.rb

```diff
@@ -51,6 +50,11 @@
     "NEEDS_LIBICONV"       => "1",
     "NO_GETTEXT"           => "1",
     "NO_PYTHON"            => "1",
+    # Disabling perl - we don't currently need any of the provided
+    # functionality: https://github.com/git/git/blob/master/INSTALL#L102-L109
```

Comment:
Maybe use this link which will be more "perma" ? https://github.com/git/git/blob/563e38491eaee6e02643a22c9503d4f774d6c5be/INSTALL#L102-L109